hack/local-up-cluster.sh

```diff
@@ -565,13 +565,23 @@ function start_apiserver {
       cloud_config_arg="--cloud-provider=external"
     fi
 
+    cat <<EOF > /tmp/kube-audit-policy-file
```

Comment:
only do this if `$AUDIT_POLICY_FILE` is unset, and set AUDIT_POLICY_FILE to /tmp/kube-audit-policy-file here, and use `--audit-policy-file="${AUDIT_POLICY_FILE}" \` below?

Reply:
I wish we weren't hardcoding to `/tmp`, but it's all over the file so it's not exactly going to break anyone.

Reply:
I had the same thought, and the same follow up response. I wouldn't do something different just for this file